x-pack/test/plugin_api_integration/test_suites/task_manager/migrations.ts

```diff
@@ -76,5 +80,29 @@ export default function createGetTests({ getService }: FtrProviderContext) {
         )}"}`
       );
     });
+
+    it('8.2.0 migrates alerting tasks that has no schedule.interval', async () => {
+      const searchResult: TransportResult<
+        estypes.SearchResponse<{ task: ConcreteTaskInstance }>,
+        unknown
+      > = await es.search(
+        {
+          index: '.kibana_task_manager',
+          body: {
+            query: {
+              term: {
+                _id: 'task:d33d7590-8377-11ec-8c11-2dfe94229b95',
+              },
+            },
+          },
+        },
+        { meta: true }
+      );
+      expect(searchResult.statusCode).to.equal(200);
+      expect((searchResult.body.hits.total as estypes.SearchTotalHits).value).to.equal(1);
+      const hit = searchResult.body.hits.hits[0];
+      expect(hit!._source!.task.attempts).to.be(0);
+      expect(hit!._source!.task.status).to.be(TaskStatus.Idle);
```

Comment:
Looks like this test might be flaky because task manager will try to claim the task on startup and then the status will be `claiming`. You can add a `retry.try` or set the `runAt` in the task document to super far in the future and make it a tomorrow problem :)

Reply:
Setting runAt to a late date sounds like a better option. So the test runner wouldn't be overwhelmed :)